Math/src/main/java/org/tribuo/math/la/DenseMatrix.java

```diff
@@ -47,6 +52,10 @@ public DenseMatrix(int dim1, int dim2) {
         this.numElements = dim1*dim2;
     }
 
+    /**
+     * Copies the supplied matrix.
+     * @param other The matrix to copy.
+     */
```

Comment:
Just curious why we're not using arraycopy in this constructor?

Reply:
Because it's not final and `ShrinkingMatrix` and `AdaGradRDAMatrix` both subclass `DenseMatrix` and mess with the get method to apply a transformation as part of the regularisation during training. I guess I could check if it's only a DenseMatrix without any other classes and then do an arraycopy, falling back to this code otherwise, but it doesn't seem worth it at the moment.